### ISSUE TITLE
cli: There should also be a message about possibly locked device

### ISSUE THREAD

**Issue by vi** [whoozle/android-file-transfer-linux#74]
GUI version shows "No MTP storage found, you device might be locked"

CLI version just prints:

```
HUAWEI H60-L12> ls
error: Get: invalid response code InvalidStorageID (0x2008)
```

I think it should print a message about locked device as well.

**Comment by vi** [comment]
`aft-mtp-mount` should also warn. Now it just shows empty directory when the device is locked and fails to output anything meaningful even with `-o debug`.

**Comment by whoozle** [comment]
fixed in e129a00be468a5dfbf74087d095ed2f478ddceb4
